src/commands/data/schemas/delete.ts

```diff
@@ -10,13 +11,14 @@ export const builder = (yargs: any) => epilogue(yargs).option('id', {
 });
 
 export const handler = async ({ sdk, id }) => {
-  const disableResponse = await sdk.raw.post(`/data/v1/${id}/disable`);
-  if (disableResponse.data?.affectedRecords !== 1) {
+  const disableResponse = await schemaRepository.disable(sdk, id);
```

Comment:
I'm not mad this is changed, but I definitely wasn't expecting to see this here 😅

Reply:
Hehe me neither 😅 but it was really hard to test this otherwise 😬